### PR TITLE
Add integration test for groups API

### LINKED ISSUE #7: Set up servers-side testing framework
Done when: server-side code can be unit tested and integration tested with the test database. Recommend using Jest for a testing framework but open to other options.

Add test runs to CI: https://github.com/distributeaid/shipment-tracker/blob/saga/.github/workflows/ci.yml#L33-L34

### PR DESCRIPTION
**Pull request by bion**
Closes #7 

- [x] passing API integration tests that exercise the database
- [x] tests added to CI
- [x] make SQL logging configurable, disabled by default
- [x] set up pg on CI

### CODE REVIEW

File: src/sequelize.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { Sequelize } from 'sequelize-typescript'
 
-const env = process.env.NODE_ENV || 'development'
+const env = process.env.DB_ENV || process.env.NODE_ENV || 'development'
```
Comment (andrewjacobson-outreach) [inline src/sequelize.ts:6]: Haven't seen `DB_ENV` used before. When does this get set instead of `NODE_ENV`?

**Review by andrewjacobson-outreach — APPROVED** [review]
LGTM!

File: src/sequelize.ts (hunk 1 above)
Comment (bion) [reply src/sequelize.ts:6]: It's just for CI since the DB connection is slightly different for CI vs local test. I'll add a comment to clarify.

Comment (andrewjacobson-outreach) [reply src/sequelize.ts:6]: Ah got it. Sounds good 👍

**Pull request merged**